charts/capsule-proxy/values.yaml

```diff
@@ -67,17 +67,19 @@ options:
   rolebindingsResyncPeriod: 10h
 
 certManager:
-  # -- Set if the cert manager will generate self-signed SSL certificates
+  # -- Set if the cert manager will generate SSL certificates (self-signed or CA-signed)
   generateCertificates: false
   externalCA:
+    # -- Set if want cert manager to sign certificates with an external CA
     enabled: false
     # secret containing the CA cert and private key of the external CA in the correct cert-manager format as per https://cert-manager.io/docs/configuration/ca/#deployment
     secretName: ""
   issuer:
-    # -- Set if the cert manager will generate either self-signed or CA signed SSL certificates. Its value will be either Issuer or ClusterIssuer
+    # -- Set if the cert manager will generate either self-signed or CA signed SSL certificates.
+    # -- Its value will be either Issuer or ClusterIssuer
     kind: Issuer # Issuer or ClusterIssuer
     # -- Set the name of the ClusterIssuer if issuer kind is ClusterIssuer and if cert manager will generate CA signed SSL certificates
-    name: "" #  Name of the ClusterIssuer
+    clusterIssuerName: "" #  Name of the ClusterIssuer
```

Comment:
This would be a breaking change and would require bumping up the minor version.

Is it required?

Reply:
I see, I think it's not required 👍🏻

Reply:
Just reverted @pro